scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -382,12 +382,22 @@
             :to-make-room-for pending-job)]
     [state nil]))
 
+(defn reserve-hosts! [state preemption-decisions]
+  (let [multiple-tasks (filter #(< 1 (count (:task %))) preemption-decisions)
+        reservations (into {} (map (fn [d] [(:job/uuid (:to-make-room-for d))
+                                            (:hostname d)])
+                                   multiple-tasks))]
+    (swap! state (fn [{:keys [jobs-launched]}]
```

Comment:
Should this be `launched-jobs`? Also can we please rename to `launched-job-uuids`?

Reply:
Done